src/storekit.cs

```diff
@@ -760,7 +793,7 @@ interface SKProductSubscriptionPeriod {
 		SKProductPeriodUnit Unit { get; }
 	}
 
-	[iOS (11,2), TV (11,2), Mac (10,13,2)]
+	[Watch (4,2), iOS (11,2), TV (11,2), Mac (10,13,2)]
```

Comment:
Should this be `Watch(6,2)`? Apple docs say that it's been available since `Watch(4,2)`, but we are only adding it now...

Reply:
likely a typo from Apple
the framework was not available before 6.2 - so the API was not

Reply:
it needs to be tested to make sure, if we did not add it there most be a reason. Add it to 6,0 which is safer than adding an API to a later version and then not being able to remove it.